opendbc/car/rivian/values.py

```diff
@@ -0,0 +1,63 @@
+from enum import IntFlag
+from opendbc.car.structs import CarParams
+from opendbc.car import Bus, structs
+from opendbc.car import CarSpecs, PlatformConfig, Platforms
+from opendbc.car.docs_definitions import CarDocs
+from opendbc.car.fw_query_definitions import FwQueryConfig, Request, StdQueries
+
+Ecu = CarParams.Ecu
+
+
+class CAR(Platforms):
+  RIVIAN_R1S_GEN1 = PlatformConfig(
+    [CarDocs("Rivian R1S 2022-24", "All")],
+    CarSpecs(mass=3206., wheelbase=3.08, steerRatio=15.2),
+    {Bus.pt: 'rivian_can'}
+  )
+
+# TODO ↓
+FW_QUERY_CONFIG = FwQueryConfig(
+  requests=[
+    Request(
+      [StdQueries.TESTER_PRESENT_REQUEST, StdQueries.SUPPLIER_SOFTWARE_VERSION_REQUEST],
+      [StdQueries.TESTER_PRESENT_RESPONSE, StdQueries.SUPPLIER_SOFTWARE_VERSION_RESPONSE],
+      whitelist_ecus=[Ecu.eps],
+      rx_offset=0x08,
+      bus=0,
+    ),
+    Request(
+      [StdQueries.TESTER_PRESENT_REQUEST, StdQueries.UDS_VERSION_REQUEST],
+      [StdQueries.TESTER_PRESENT_RESPONSE, StdQueries.UDS_VERSION_RESPONSE],
+      whitelist_ecus=[Ecu.engine],
+      rx_offset=0x10,
```

Comment:
```suggestion
      rx_offset=-0x6e,
```

try this for the engine

Reply:
The values are just a copy and paste from my Tesla port.